internal/cache/svg.go

```diff
@@ -20,11 +20,15 @@ func GetSvg(name string, w int, h int) *image.NRGBA {
 	return sinfo.pix
 }
 
-// ResetSvg clears all the svg cache map
-func ResetSvg() {
+// ResetThemeCaches clears all the svg and text size cache maps
+func ResetThemeCaches() {
```

Comment:
Hmm, maybe it should be better to move this function to base.go? To keep here only the svg related functions.

Reply:
That was my first choice, but that file relates to a graphical cache used for painting and renderers etc so it didn't seem to fit. I will try looking at it again.

Reply:
Seems like it wasn't the clean cut I thought. I moved it in and moved some svg stuff out of base.go so it made more sense.